pages/docs/project-management/collections.mdx

```diff
@@ -7,21 +7,24 @@ description: Collections allow you to group related saved explorations dashboard
 Collections allow you to group related saved explorations, dashboards, or other collections
 together to allow your organization to more easily find and share related data.
 
-You can see all your collections on the [collections page](https://hashboard.com/app/p/collections).
-From this page, you can also create a new collection.
+You can navigate to a collection by clicking on it in the left sidebar or searching for
```

Comment:
we didn't get rid of the collections page, right? why remove the link?

Reply:
There is no clear way to link to that page anymore and I think it was discussed that we would remove this page.